presto-tests/src/test/java/com/facebook/presto/tests/TestQueryManager.java

```diff
@@ -111,26 +130,41 @@ public void testFailQueryPrerun()
                 "SELECT * FROM lineitem")
                 .get();
 
+        assertNotEquals(dispatchManager.getStats().getQueuedQueries(), 0L);
```

Comment:
This seems like a weird test assertion; I think you're trying to cause a failure if the dispatchManager behaves differently than expected. If you could use the version of assertNotEquals that takes a message to annotate that, I think it would be helpful.